ir_datasets/formats/argsme.py

```diff
@@ -0,0 +1,412 @@
+from datetime import datetime
+from enum import Enum
+from pathlib import Path
+from re import sub
+from typing import NamedTuple, List, Optional
+
+from ijson import items
```

Comment:
Can you change this so the library is only loaded when it's needed using `lazy_libs`? This keeps the loading time of the package low (particularly important when using the CLI).

```python
ijson = ir_datasets.lazy_libs.ijson()
```

Example: https://github.com/allenai/ir_datasets/blob/master/ir_datasets/datasets/dpr_w100.py#L43